packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -30,6 +30,10 @@ const EdgeInsets _kFloatingCursorSizeIncrease = EdgeInsets.symmetric(horizontal:
 // The corner radius of the floating cursor in pixels.
 const Radius _kFloatingCursorRadius = Radius.circular(1.0);
 
+// Represents the shortest distance required when both the floating cursor
+// and regular cursor are present.
```

Comment:
nit: maybe add a bit more context about the behavior? Something like in UITextField when floating cursor is activated the regular cursor won't be visible unless it's too far away from the regular one?